tests/acceptance/components/flexberry-objectlistview/folv-column-width-customization-test.js

```diff
@@ -0,0 +1,117 @@
+import Ember from 'ember';
+import { executeTest } from './execute-folv-test';
+import { loadingList} from './folv-tests-functions';
+
+executeTest('column width custom', (store, assert, app) => {
+
+  let path = 'i-i-s-caseberry-logging-objects-application-log-l';
+  Ember.run(() => {
+    let done = assert.async();
+    visit(path);
+    andThen(function() {
+      assert.equal(currentPath(), path);
+      let $defaultConfig = Ember.$('.remove.circle.icon');
+      loadingList($defaultConfig[0], '.object-list-view-container', 'table.object-list-view tbody tr').then(($list) => {
+        assert.ok($list);
+        let $configButton = Ember.$('.ui.button');
+        let done1 = assert.async();
+
+        // Loading config window
+        loadingList($configButton[1], '.object-list-view-container', 'table.object-list-view tbody tr').then(($list) => {
+          assert.ok($list);
+
+          // Retrieve component & it's inner <input>.
+          let $component = Ember.$('.flexberry-checkbox.ui.checkbox');
+          let $checkboxInput = $component.children('input');
+
+          assert.equal($($component).hasClass('checked'), false, 'Width config is OFF');
```

Comment:
You can simply use the function [`assert.notOk`](https://api.qunitjs.com/assert/notOk) instead of comparing with `false`.